qiime2_TagSequencingReport.Rmd

```diff
@@ -0,0 +1,383 @@
+  filter(!is.na(time_collected)) %>%
+  ggplot(aes(x=eval(parse(text=color_by)), y=eval(parse(text=alpha_measure)), color=eval(parse(text=color_by)))) +
+  geom_boxplot() +
+  labs(y=alpha_measure, x=color_by, color=color_by) +
+  theme_bw() +
+  theme(axis.text.x=element_text(angle=-25, hjust= .1)) +
+  scale_color_brewer(palette = "Set2")
+```
+
+\newpage 
+
+# Beta diversity
+
+Similarity between samples was assessed by unweighted and weighted UniFrac distances. 
+
+## Unweighted UniFrac distances
+
+```{r}
+dist_type <- "uu" ## unweighted or weighted
+if (dist_type=="uu") {
+  dist_fp <- uu_fp
+  dist_name <- "unweighted UniFrac distances"
+} else if (dist_type=="wu") {
+  dist_fp <- wu_fp
+  dist_name <- "weighted UniFrac distances"
+}
+```
+
+The `r dist_name` was employed to compare the species composition of the samples to each other. We used a method of ordination called Principal Coordinates Analysis to select the best 2D coordinate system for display. The percentage in each axis represents the proportion of the total variance captured along the axis. 
+
+### PCoA plot
+
+```{r, fig.width=6, fig.height=5, echo=FALSE, warning=FALSE}
+s_toPlot <- s[s$Read_Counts > min_reads, ]
+s_toPlot$Keep <- ifelse(grepl("geneblock|freewater|control", s_toPlot$SampleID, ignore.case = TRUE), FALSE, TRUE)
+
+dist_in <- read_qiime_distmat(dist_fp)
+dist_in <- dist_subset(dist_in, s_toPlot$SampleID[s_toPlot$Keep])
+pcoa_title <- paste0("PCoA plot based on ", dist_name)
+pcoa_plot <- make_pcoa_plot(dist_in, s_toPlot[s_toPlot$Keep,], color_by=color_by, shape_by=NULL, title = pcoa_title)
+plot(pcoa_plot)
+```
+
+\newpage
+
+### UPGMA clustering 
+
+The following plot shows sample clustering based on `r dist_name`. We have used a method of hierarchical clustering called "average-linkage" or UPGMA. At the bottom of the dendrogram, all samples start out in their own group. Moving up the dendrogram, samples accumulate into clusters if the average (mean) distance between all samples is below the indicated value.
+
+```{r, fig.width=12, fig.height=5, echo=FALSE}
+hc = hclust(dist_in, method="average")
+plot(hc, main=paste0("UPGMA linkage clustergram based on\n", dist_name), xlab="", ylab="distance", sub = "")
+```
+
+\newpage
+
+## Weighted UniFrac distances
+
+```{r}
+dist_type <- "wu" ## unweighted or weighted
+if (dist_type=="uu") {
+  dist_fp <- uu_fp
+  dist_name <- "unweighted UniFrac distances"
+} else if (dist_type=="wu") {
+  dist_fp <- wu_fp
+  dist_name <- "weighted UniFrac distances"
+}
+```
+
+The `r dist_name` was employed to compare the species composition of the samples to each other. We used a method of ordination called Principal Coordinates Analysis to select the best 2D coordinate system for display. The percentage in each axis represents the proportion of the total variance captured along the axis. 
+
+### PCoA plot
+
+```{r, fig.width=6, fig.height=5, echo=FALSE, warning=FALSE}
+s_toPlot <- s[s$Read_Counts > min_reads, ]
```

Comment:
You can filter by the variable Keep because you have already made this filtering and generated the column on line 224

Reply:
that's correct, will replace using Keep. Thanks!